src/packet/vp8_contiguity.rs

```diff
@@ -2,31 +2,29 @@ use super::Vp8CodecExtra;
 use std::collections::VecDeque;
 
 #[derive(Debug)]
-pub struct Vp8Contiguity {
+pub(crate) struct FrameContiguityState {
```

Comment:
Should we maybe move `FrameContiguityState` to packet/contiguity.rs since it's shared for both vp8 and vp9?